packages/components/src/public/QueryModel/ChartMenu.tsx

```diff
@@ -75,12 +76,13 @@ export class ChartMenu extends PureComponent<Props> {
         const disabled = isLoading || isLoadingCharts || noCharts || hasError;
         const selectedChart = charts?.find(chart => chart.reportId === selectedReportId);
         const showChartModal = queryInfo !== undefined && selectedChart !== undefined;
+        const _hideEmptyChartMenu = getQueryMetadata().get('hideEmptyChartMenu', hideEmptyChartMenu);
```

Comment:
Personally, I would probably inline this instead of using the underscore const, but that's probably just me.